doc/source/cluster/kubernetes/user-guides/gcp-gke-gpu-cluster.md

```diff
@@ -2,6 +2,8 @@
 
 # Start Google Cloud GKE Cluster with GPUs for KubeRay
 
+See <https://cloud.google.com/kubernetes-engine/docs/how-to/gpus> for full details, or continue reading for a quick start.
```

Comment:
What does `<https://cloud.google.com/kubernetes-engine/docs/how-to/gpus>` look like after rendering?

Reply:
<img width="758" alt="Screenshot 2023-08-29 at 8 17 00 AM" src="https://github.com/ray-project/ray/assets/5459654/5400248b-d75f-44f1-89c8-ade09ad7f606">
My markdown linter complains if I don't add `<` and `>`